src/main/kotlin/org/jitsi/nlj/rtcp/RtcpRrGenerator.kt

```diff
@@ -115,7 +115,7 @@ class RtcpRrGenerator(
             when (packets.size) {
                 0 -> {}
                 1 -> rtcpSender(packets[1])
-                else -> rtcpSender(CompoundRtcpPacket(packets))
+                else -> rtcpSender(CompoundRtcpPacket(*packets.toTypedArray()))
```

Comment:
Hmm, this makes the vararg less appealing.  Maybe we should do the list (or support both).